### DISCUSSION TITLE
How do I make Open-Shell Start Button work when taskbar is hidden?

### DISCUSSION THREAD

**Discussion by Hambruhger** [Open-Shell/Open-Shell-Menu#1596] [Q&A]
So, I use RetroBar as my taskbar. Only problem is that it makes a gap between the windows and the taskbar. A workaround is to hide the taskbar, fixing the problem. I also use Open-Shell, and this creates another problem: the Open-Shell Start Button gets hidden with the taskbar, making it impossible to get that Windows XP taskbar feel. Anyone know how to fix this?

**Comment by ge0rdi** [comment]
Doesn't the taskbar automatically un-hide when you go with mouse cursor to the bottom of your screen (or where your taskbar is located)?
Or if you press Win key?

**Reply by Hambruhger** [reply]
well yes, but retrobar completely hides the taskbar

**Comment by Imold** [comment]
If you're on Windows 11 instead of using the "automatically hide the taskbar" setting, install [Windhawk](https://windhawk.net/) and use the ["Taskbar height and icon size" mod](https://windhawk.net/mods/taskbar-icon-size) to shrink the Windows taskbar to the same height as retrobar.
